analyzer/src/funTest/kotlin/BabelTest.kt

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2017-2018 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package com.here.ort.analyzer
+
+import com.here.ort.analyzer.managers.NPM
+import com.here.ort.downloader.VersionControlSystem
+import com.here.ort.utils.yamlMapper
+
+import io.kotlintest.TestCaseContext
+import io.kotlintest.matchers.shouldBe
+import io.kotlintest.specs.WordSpec
+
+import java.io.File
+
+class BabelTest : WordSpec() {
+    private val projectDir = File("src/funTest/assets/projects/synthetic/npm-babel")
+    private val vcsDir = VersionControlSystem.forDirectory(projectDir)!!
+    private val vcsRevision = vcsDir.getRevision()
+
+    @Suppress("CatchException")
+    override fun interceptTestCase(context: TestCaseContext, test: () -> Unit) {
+        try {
+            super.interceptTestCase(context, test)
+        } catch (exception: Exception) {
```

Comment:
Why not use a `finally` block instead of `catch-throw`?

Reply:
Good point. I've copied this code from `NpmTest.kt`. Will make a follow-up PR to change this in both files.

Reply:
Ok.